config/v1/types_infrastructure.go

```diff
@@ -339,6 +339,15 @@ type AWSPlatformSpec struct {
 	// There must be only one ServiceEndpoint for a service.
 	// +optional
 	ServiceEndpoints []AWSServiceEndpoint `json:"serviceEndpoints,omitempty"`
+
+	// ResourceTags is a list of additional tags to apply to AWS resources created for the cluster.
+	// See https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html for information on tagging AWS resources.
+	// AWS supports a maximum of 50 tags per resource. OpenShift reserves 25 tags for its use, leaving 25 tags
+	// available for the user.
+	// While ResourceTags field is mutable, items can not be removed.
```

Comment:
Is there anything that will force items not to be removed, or can they actually be removed and that just won't be reflected by the tags on the AWS resources?

If it's the latter, we may want to adjust the language here to make that more obvious

Reply:
Nothing in the schema definition. I intend to use dynamic admission control for this.

Reply:
Is it better to add link to enhancement that explains different tag removal behavours?

Reply:
It's better to have this directly in the godoc. Content in the godoc should be customer focused as it is what they will see if they `oc describe` a resoucre. We should try to make sure there's a succinct description of field behaviours on all fields within the API